src/itwinai/torch/distributed.py

```diff
@@ -613,24 +564,28 @@ def init(self) -> None:
         import deepspeed
 
         self.deepspeed = deepspeed
-        if not distributed_resources_available():
-            raise RuntimeError("Trying to run distributed on insufficient resources.")
+        force_dist_env = int(os.environ.get('ITWINAI_FORCE_DIST', '0'))
+        if not distributed_resources_available() and not force_dist_env:
```

Comment:
Maybe to make this a little more neat we could move the `force_dist_env = int(os.environ.get('ITWINAI_FORCE_DIST', '0'))` to the `distributed_resources_available()` function so as to avoid duplication, and instead call it something like `should_run_distributed()`. Just a design preference, both would work obviously

Reply:
good find. Fixed it